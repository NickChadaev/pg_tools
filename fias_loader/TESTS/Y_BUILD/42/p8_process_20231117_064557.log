2023-11-17 06:45:57 home/n.chadaev@abrr.local
127.0.0.1 5432 unsi_test_42 postgres /home/n.chadaev@abrr.local/Y_BUILD/42/stage_gar_c_42.csv 2023-11-13
--------------------------------------------------------
[06:45:57] ... Кемеровская обл., id_region: 41, Execute: /home/n.chadaev@abrr.local/Y_BUILD/42/stage_gar_c_42.csv, process_id:25345
[06:45:57] ... Start process
[06:45:57] ...  -- Version
[06:45:58] ...  -- Очистка от данных
[06:45:58] ...  -- Убираем индексы
[06:45:58] ...  -- Set UNLOGGED
[06:45:58] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_APARTMENT_TYPES_20231113_78e8cff6-619c-4b2e-806f-6c3c18c41d8c.XML
[06:45:58] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDR_OBJ_TYPES_20231113_15f2ec37-c925-4252-b49a-5473d975a16f.XML
[06:45:58] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ROOM_TYPES_20231113_32952955-cd74-40c1-bc84-06d94fe9559d.XML
[06:45:58] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OPERATION_TYPES_20231113_5151274c-91f4-4286-aaff-fc37112b660a.XML
[06:45:58] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_PARAM_TYPES_20231113_fe767969-20f3-4044-9ef6-67b4cb2881ce.XML
[06:45:58] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_HOUSE_TYPES_20231113_507d1000-4460-4d2f-a5b1-2ab5b9cd8a26.XML
[06:45:58] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDHOUSE_TYPES_20231113_d99b31fe-7b05-400a-ac92-23919e58da07.XML
[06:45:58] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OBJECT_LEVELS_20231113_b5fa7b3d-ea26-40c9-8b3a-9b986261ea66.XML
[06:45:58] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_TYPES_20231113_b8068ed2-dd18-4365-abb7-3dbd98698c35.XML
[06:45:58] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_KINDS_20231113_2f0f38c6-d598-4fd7-b325-2aa6aca29ed2.XML
[06:45:58] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/42/AS_REESTR_OBJECTS_20231113_11c3bdb0-9b9f-4e41-a1a1-689fed63e682.XML
[07:07:47] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/42/AS_ADDR_OBJ_20231113_d66e97e3-7bae-4533-b1ca-22706b4955f7.XML
[07:08:17] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/42/AS_HOUSES_20231113_9b86e1a0-2043-41ca-baf4-31486ca96f50.XML
[07:21:42] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/42/AS_STEADS_20231113_74492beb-d23b-4aaa-b0df-df7084fd2144.XML
[07:23:32] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/42/AS_APARTMENTS_20231113_71b8e035-67e1-42ce-9153-31a4c92e27a2.XML
[07:35:47] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/42/AS_ROOMS_20231113_6e3eda56-9b10-48c7-9785-8038a164864f.XML
[07:35:49] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/42/AS_CARPLACES_20231113_062f36c0-762e-4fc5-ad21-c8265dc212c1.XML
[07:35:50] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/42/AS_ADM_HIERARCHY_20231113_40d39af3-109d-4e53-9088-fecdc93249a4.XML
[08:03:30] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/42/AS_MUN_HIERARCHY_20231113_d72c9683-2991-4b23-88b2-6e22c6df80e7.XML
[08:34:11] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/42/AS_CHANGE_HISTORY_20231113_d799f8af-abf0-4215-a459-1d69aac49a77.XML
[09:38:51] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/42/AS_ADDR_OBJ_DIVISION_20231113_bc3528ac-ba1a-4ee0-ba8c-4c2e4a7e33a6.XML
[09:38:51] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/42/AS_ADDR_OBJ_PARAMS_20231113_5ccd3bd9-a679-45e6-860c-a52e90982281.XML
[09:43:35] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/42/AS_STEADS_PARAMS_20231113_15a76cfd-794c-412f-8971-aef588abd599.XML
[10:03:43] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/42/AS_HOUSES_PARAMS_20231113_61b3fccb-3341-4fa0-bbee-177a50d6cf80.XML
[11:27:18] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/42/AS_APARTMENTS_PARAMS_20231113_b324103d-50eb-492b-ac08-989053ec52ee.XML
[11:46:31] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/42/AS_ROOMS_PARAMS_20231113_6287dd99-9b15-44ee-a298-da6b3952df31.XML
[11:46:34] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/42/AS_CARPLACES_PARAMS_20231113_a37e267b-cbd9-4c7b-8bc1-e544d2d2c3c2.XML
[11:46:34] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/42/AS_NORMATIVE_DOCS_20231113_6fbcb46e-5f9f-4ef5-be55-d64f28c5dd2f.XML
[11:49:17] ...  -- Data finish
[11:49:17] ...  -- Set LOGGED
[11:50:01] ... Stop process
--------------------------------------------------------
