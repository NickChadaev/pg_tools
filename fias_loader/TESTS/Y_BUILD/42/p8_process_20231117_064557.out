 save_gar_version 
------------------
               81
(1 row)

CALL
CALL
CALL
 set_gar_dt_create 
-------------------
                81
(1 row)

CALL
gar_fias.as_apartment_type: 13
gar_fias.as_addr_obj_type: 421
gar_fias.as_room_type: 3
gar_fias.as_operation_type: 27
gar_fias.as_param_type: 20
gar_fias.as_house_type: 14
gar_fias.as_add_house_type: 4
gar_fias.as_object_level: 17
gar_fias.as_norm_docs_types: 25
gar_fias.as_norm_docs_kinds: 4
gar_fias.as_reestr_objects: 1754009
gar_fias.as_addr_obj: 34015
gar_fias.as_houses: 1001898
gar_fias.as_steads: 153635
gar_fias.as_apartments: 992629
gar_fias.as_rooms: 3039
gar_fias.as_carplaces: 460
gar_fias.as_adm_hierarchy: 2185000
gar_fias.as_mun_hierarchy: 2460373
gar_fias.as_change_history: 5075616
gar_fias.as_addr_obj_division: 63
gar_fias.as_addr_obj_params: 346866
gar_fias.as_steads_params: 1520103
gar_fias.as_houses_params: 6733358
gar_fias.as_apartments_params: 1550341
gar_fias.as_rooms_params: 3728
gar_fias.as_carplaces_params: 61
gar_fias.as_normative_docs: 211500
